recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
   run:
     - python <3.7
     - astropy
-    - six
+    - gsl
```

Comment:
gsl in run but not host is really weird. is the library linking to gsl? Why would it only be required when running?

Reply:
Ah, oops!